### ISSUE TITLE
update img alt accname calculation to match aria-label

### ISSUE THREAD

**Issue by pkra** [w3c/html-aam#595]
This is a continuation of #238 which had been closed in favor of https://github.com/w3c/accname/issues/55 and other work in accname itself. As a result, accname nowadays treats whitespace (properly defined) like the empty string.

I believe we overlooked a matching change here though. 

While 4.1.10 (img Element Accessible Name Computation) now indirectly treats whitespace-only aria-label as having no name (via accname), the alt attribute still reads

> An img with an alt attribute whose value is the empty string is mapped to the presentation role. It has no accessible name.

In other words, `alt=" "` would be treated differently from `aria-label=" "`.  I believe implementations treat them the same (there's no accessible name).
